media/css/devreg/payments.styl

```diff
@@ -697,7 +697,7 @@ label .region {
 }
 
 // India (in) is in quotes since `in` is a reserved word.
-for region in ar bd br bw ci cl cm cn co cr cz de ec eg es fr gr gt gw hu 'in' it jo jp lt me mg ml mm mu mx ne ni pa pe ph pl rs ru sn sv tn tz uk us uy ve vu za restofworld
+for region in ar bd br bw cf ci cl cm cn co cr cz de ec eg es fr gn gr gt gw hu 'in' it jo jp ke lt me mg ml mm mu mx ne ni pa pe ph pl rs ru sn sv tn tz uk us uy ve vu za restofworld
```

Comment:
I wanted to do that, but zamboni only imports marketplace-constants as a python package, it doesn't handle front-end dependencies at all, so that would require some changes in the way it's built. I'll file a bug though.